[ENHANCEMENT] Add application on F-Droid

Hello,

In the scope of releasing the application in a full open-source environment, is it plan to release the app on the F-Droid alternative store ? https://f-droid.org/

```
Process: com.flipperdevices.app, PID: 23835
                                                java.lang.RuntimeException: Unable to get provider io.sentry.android.core.SentryInitProvider: java.lang.IllegalArgumentException: DSN is required. Use empty string to disable SDK.
                                                	at android.app.ActivityThread.installProvider(ActivityThread.java:7493)
                                                	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6999)
                                                	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6770)
                                                	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
                                                	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2134)
                                                	at android.os.Handler.dispatchMessage(Handler.java:106)
                                                	at android.os.Looper.loopOnce(Looper.java:201)
                                                	at android.os.Looper.loop(Looper.java:288)
                                                	at android.app.ActivityThread.main(ActivityThread.java:7898)
                                                	at java.lang.reflect.Method.invoke(Native Method)
                                                	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                                                	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
                                                Caused by: java.lang.IllegalArgumentException: DSN is required. Use empty string to disable SDK.
                                                	at ko.n1.d(Sentry.java:824)
                                                	at ko.n1.c(Sentry.java:49)
                                                	at io.sentry.android.core.SentryInitProvider.onCreate(SentryInitProvider.java:113)
                                                	at android.content.ContentProvider.attachInfo(ContentProvider.java:2451)
                                                	at android.content.ContentProvider.attachInfo(ContentProvider.java:2421)
                                                	at io.sentry.android.core.SentryInitProvider.attachInfo(SentryInitProvider.java:15)
                                                	at android.app.ActivityThread.installProvider(ActivityThread.java:7488)
                                                	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6999) 
                                                	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6770) 
                                                	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0) 
                                                	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2134) 
                                                	at android.os.Handler.dispatchMessage(Handler.java:106) 
                                                	at android.os.Looper.loopOnce(Looper.java:201) 
                                                	at android.os.Looper.loop(Looper.java:288) 
                                                	at android.app.ActivityThread.main(ActivityThread.java:7898) 
                                                	at java.lang.reflect.Method.invoke(Native Method) 
                                                	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
                                                	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936) 
```

--- a/README.md
+++ b/README.md
@@ -5,6 +5,16 @@ Mobile app to rule all Flipper's family
 ![dolphin-dark](https://user-images.githubusercontent.com/5871715/163869541-33904d20-7684-4891-abf9-be0a0c1afe71.png#gh-dark-mode-only)
 ![dolphin-light](https://user-images.githubusercontent.com/5871715/163869555-fe5b029d-c4bd-4a29-92b7-fc9c79505106.png#gh-light-mode-only)
 
+## Download
+
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
+     height="80">](https://f-droid.org/packages/com.flipperdevices.app/)
+[<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png"
+     alt="Get it on Google Play"
+     height="80">](https://play.google.com/store/apps/details?id=com.flipperdevices.app)
+
+or get the app from the [Releases Section](https://github.com/flipperdevices/Flipper-Android-App/releases/latest).
 
 ## Module arch
 

